# The action goal
pose set_pose
---
# Action result
pose final_pose
---
# Action feedback
pose current_pose